docs/README.md

```diff
@@ -1,3 +1,83 @@
 # K8ssandra.io - Website & Documentation Repository
 
-See the common docs README for more information [here](https://github.com/k8ssandra/k8ssandra/tree/docs#readme).
+This project is built with the [Hugo](https://gohugo.io/) static site generator and the [Docsy](https://github.com/google/docsy) theme.
+
+The documentation produced from this repository can be viewed at:
+
+Development Version -- https://docs.k8ssandra-dev.io
+
+Production Version -- https://docs.k8ssandra.io
+
+## Dependencies
+
+This project requires Node.js and NPM to be installed locally.  All other dependencies are provided via NPM.
+
+The latest version of Node/NPM should work, the current automation uses Node 14.
+
+You can install Node in a myriad of ways, check [here](https://nodejs.org/en/) for more information.
+
+
+## Development
+
+Docsy is added as hugo module. No external installations are required for docsy.
+### Install dependencies
+
+From the `/docs` directory, run
+
+```
+npm install
+```
+
+This command will install the project dependencies, such as hugo-extended.
+
+### Scripts
+
+There are a number of utility scripts provided in package.json that can be executed for various purposes:
+
+#### Start the Hugo server for local development
+
+```
+npm run start
+```
+
+This provides a live-refresh server that will automatically load any changes made in the source code.  
+
+The local server will be available at http://localhost:1313/.
+
+#### Use Hugo to build the site
+
+```
+npm run build:dev
+```
+
+or
+
+```
+npm run build:prod
+```
```

Comment:
In which cases should we use one or the other?

Reply:
In the previous documentation environments are set for hugo in package.json. I am not sure if these would be used in future. Just kept in case. its not mandatory steps.

Reply:
removed these